roles/olm/files/20-aggregated-edit.clusterrole.yaml

```diff
@@ -10,5 +10,5 @@ metadata:
     rbac.authorization.k8s.io/aggregate-to-edit: "true"
 rules:
 - apiGroups: ["operators.coreos.com"]
-  resources: ["*"]
-  verbs: ["*"]
\ No newline at end of file
+  resources: ["clusterserviceversions", "catalogsources", "installplans", "subscriptions"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```

Comment:
I have not reviewed the security implications of this, but `edit` should not include the verbs that are already in `view`.  View is automatically aggregated into edit.